VisualPinball.Unity/VisualPinball.Unity.Editor/Utils/HandlesUtils.cs

```diff
@@ -0,0 +1,74 @@
+﻿using UnityEditor;
+using UnityEngine;
+using VisualPinball.Unity.VPT;
+
+namespace VisualPinball.Unity.Editor.Utils
+{
+	class HandlesUtils
+	{
+		public delegate void OnPositionChange(Vector3 newPosition, params (string,object)[] plist);
+
+		public static void HandlePosition(Vector3 position, ItemDataTransformType type, Quaternion rotation, OnPositionChange onChange, params (string, object)[] plist)

```

Comment:
I'm not a fan of this loose params pattern, I'd much prefer normal params, or at the very least a strongly typed params object.

Reply:
Agreed. There's also a lot going on here, seems the only usage of `plist` is to pass it to `onChange`? Can this be simplified?